### ISSUE TITLE
Test data not downloaded

### ISSUE THREAD

**Issue by aleberti** [cta-observatory/ctapipe_io_magic#35]
I noticed that in #33 and #34 tests were not passing, the reason is that it could not download the test data (could not verify the certificate). It seems that in the past days there were changes at PIC. Updating the URL of the files should be enough.

**Comment by YoshikiOhtani** [comment]
Hm, I changed the tests script but the situation is same.

**Comment by aleberti** [comment]
Nope, not a problem related to the changes. If I run the tests offline where I already have the test data downloaded, most of them pass (I run them last week), while here no one is passing. Looking at the log of the workflow which are run automatically when pushing or when creating the scripts, the problem is clearly when downloading the data from PIC, which underwent a change in the last week. I wrote Jordi about this, it seems related to certificates, but a simple update of the certificates did not help. I wrote him again on monday, but no reply yet. I tried in different machines, but the problem is the same everywhere. So, let's see if Jordi can somehow solve the problem, what I could do on my side I did :/ of course it is a bummer because automatic tests cannot be run :(
